app/controllers/add-recipe.js

```diff
@@ -0,0 +1,34 @@
+import Ember from 'ember';
+
+export default Ember.Controller.extend({
+  newName: '',
+  newPhoto: '',
+  newServings: '',
+  newTime: '',
+  newIngredients: '',
+  newDirections: '',
+  newNotes: '',
+  newFavorite: '',
+
+  actions: {
+    createRecipe() {
+      let recipe = this.store.createRecord('recipe', {
+        // id: 1000,
+        name: this.get('newName'),
+        photo: this.get('newPhoto'),
+        servings: this.get('newServings'),
+        time: this.get('newTime'),
+        ingredients: this.get('newIngredients'),
+        directions: this.get('newDirections'),
+        notes: this.get('newNotes'),
+        favorite: this.get('newFavorite'),
+      })
+      debugger
+      recipe.save()
```

Comment:
If your server doesn't return an ID as a part of the response payload, you're gonna have a bad time. You need to either configure your server to return an ID as part of the response, or, worst case scenario, generate one client side like this: http://stackoverflow.com/questions/40628814/how-do-i-get-the-unique-id-of-ember-object-when-object-is-initialized/40629256#40629256